atom/browser/api/atom_api_session.h

```diff
@@ -38,12 +38,18 @@ class Session: public mate::TrackableObject<Session>,
                public content::DownloadManager::Observer {
  public:
   using ResolveProxyCallback = base::Callback<void(std::string)>;
+  using BlobDataCallback = base::Callback<void(const base::BinaryValue&)>;
 
   enum class CacheAction {
     CLEAR,
     STATS,
   };
 
+  enum class BlobIdType {
+    PUBLIC_URL,
```

Comment:
Only supporting UUID should be enough, I don't think users would use the API to read from a blob URL.